src/components/Scheme/SchemeContainer.tsx

```diff
@@ -112,9 +114,18 @@ const SubscribedSchemeContainer = withSubscription({
   loadingComponent: <div className={css.loading}><Loading/></div>,
   errorComponent: null,
   checkForUpdate: ["schemeId"],
-  createObservable: (props: IProps) => {
+  createObservable: async (props: IProps) => {
     const arc = getArc();
     const scheme = arc.scheme(props.schemeId);
+
+    // prime the cache: get all members fo this DAO
+    await props.daoState.dao.members({ first: 100, skip: 0 }).pipe(first()).toPromise();
```

Comment:
you have to prime the cache with the exact paginated request we will use?

Reply:
no. I'd like to just get all members in one single query. But I can't bc The Graph ppl have set a maximum request amount of 100.

Reply:
This is a graphql setting, and we will change this, so we can use a single query.